heron/schedulers/src/java/com/twitter/heron/scheduler/nomad/NomadScheduler.java

```diff
@@ -64,7 +65,12 @@
   @Override
   public void initialize(Config config, Config runtime) {
     this.localConfig = config;
-    this.clusterConfig = Config.toClusterMode(config);
+    this.clusterConfig = Config.toClusterMode(Config.newBuilder()
```

Comment:
It seems cluster config is used only for downloading topology after this change? If this is true, maybe add comment or rename variable. The variable itself gives me an impression that there are a local mode and a cluster mode.